usagov/vote-gov-drupal
# Feature/vote 1940/px design voter cards

## Jira ticket

[VOTE-1940](https://cm-jira.usa.gov/browse/VOTE-1940)

## Description

Implement PX design revisions to the feature cards. Create a new field to store an alternate icon for the voter card hover state.

## Deployment and testing

### Post-deploy steps

1. lando retune
2. cd into theme, npm run build

### QA/Testing instructions

1. Login as admin
2. Navigate to the [college student voter guide
](http://vote-gov.lndo.site/node/82/edit?destination=/admin/content) and add the two icons to the associated fields. The zip file of icons is linked in the ticket.
3. View the [Guide to Voting](http://vote-gov.lndo.site/guide-voting) page and confirm it matches the design at all breakpoints

## Checklist for the Developer




## Checklist for the Peer Reviewers
